code/modules/atmospherics/machinery.dm

```diff
@@ -1,62 +1,52 @@
-/*
-Quick overview:
-
-Pipes combine to form pipelines
-Pipelines and other atmospheric objects combine to form pipe_networks
-	Note: A single pipe_network represents a completely open space
-
-Pipes -> Pipelines
-Pipelines + Other Objects -> Pipe network
-
+ABSTRACT_TYPE(/obj/machinery/atmospherics)
+
+/**
+ * Quick overview:
+ * Pipes combine to form pipelines.
+ * Pipelines and other atmospheric objects combine to form pipe_networks.
+ * Note: A single pipe_network represents a completely open space.
+ *
+ * Pipes -> Pipelines
+ * Pipelines + Other Objects -> Pipe network
 */
-//
-obj/machinery/atmospherics
+/obj/machinery/atmospherics
 	anchored = ANCHORED
 
 	var/initialize_directions = 0
 
-	process()
-		build_network()
-		..()
-
-	// override default subscribes to be in a different process loop. that's why they don't call parent ( ..() )
-	SubscribeToProcess()
-		START_TRACKING_CAT(TR_CAT_ATMOS_MACHINES)
-
-	UnsubscribeProcess()
-		STOP_TRACKING_CAT(TR_CAT_ATMOS_MACHINES)
-
-	proc
-		network_disposing(datum/pipe_network/reference)
-			// Called by a network associated with this machine when it is being disposed
-			// This must be implemented to unhook any references to the network
-
-			return null
+/obj/machinery/atmospherics/process()
+	build_network()
+	..()
 
-		network_expand(datum/pipe_network/new_network, obj/machinery/atmospherics/pipe/reference)
-			// Check to see if should be added to network. Add self if so and adjust variables appropriately.
-			// Note don't forget to have neighbors look as well!
+// override default subscribes to be in a different process loop. that's why they don't call parent ( ..() )
```

Comment:
doc comment

Reply:
no need, doesnt do much special except subscribe to a different loop instead of the main loop. this just explains to coders why parent isnt called

Reply:
ah alrighty then